include/core.h

```diff
@@ -4,11 +4,12 @@
 #include <string.h>
 
 #include "global.h"
-#include "task.h"
 #include "sprite.h"
+#include "task.h"
 #include "tilemap.h"
-#include "animation_commands.h"
 #include "input_recorder.h"
+#include "data/sprite_data.h"
```

Comment:
Not sure about this one, these values are not supposed to be "known" to the engine, since it depends on pointer references setup on init.